logger/logger.go

```diff
@@ -17,9 +18,9 @@ func Get() *logrus.Logger {
 		logger.SetFormatter(&logrus.TextFormatter{
 			FullTimestamp: true,
 			CallerPrettyfier: func(f *runtime.Frame) (string, string) {
-				repopath := strings.Split(f.File, basePkg+"/")[1]
-				fnc := strings.Split(f.Function, basePkg+"/")[1]
-				return fmt.Sprintf("%s()", fnc), fmt.Sprintf("%s:%d", repopath, f.Line)
+				paths := strings.Split(f.File, "/")
+				return fmt.Sprintf("%s()", f.Function[len(basePkg)+1:]),
+					fmt.Sprintf("%s:%d", paths[5]+"/"+paths[6], f.Line)
```

Comment:
This is wrong. In my case, my path is `/home/hosh/Workspace/src/github.com/hoshsadiq/m3ufilter`, thus paths becomes:
```
paths = {[]string} len:10, cap:10
 0 = {string} ""
 1 = {string} "home"
 2 = {string} "hosh"
 3 = {string} "Workspace"
 4 = {string} "src"
 5 = {string} "github.com"
 6 = {string} "hoshsadiq"
 7 = {string} "m3ufilter"
 8 = {string} "server"
 9 = {string} "server.go"
```
meaning, `paths[5]+"/"+paths[6]` becomes `github.com/hoshsadiq`, which is incorrect. In addition you can't base it on the last two items.